JavaScript/js/messengerDefinition.js

```diff
@@ -1,24 +1,48 @@
-// Jacob's working on this...
+// JD's working on this...
```

Comment:
That's actually a separate issue than what I was addressing. (And I think I was mostly talking to myself.)
The part about talking about "the code" is really only about pronoun usage. It doesn't change anything else. We can still divide up the code and assign specific parts to specific people. We can still refer to "the code" I or you just worked on. (We just don't say "my code" or "your code".)